package.json

```diff
@@ -22,10 +22,15 @@
   },
   "main": "./lib/index.js",
   "types": "./types/index.d.ts",
+  "imports": {
+    "#hljs": "./build/lib/index.js",
```

Comment:
Does this work even without my other PR?  IE, if this is still all CJS?

Reply:
Yes, subpath imports are supported in Node.js v12.19.0+ and v14.6+, for both CJS and ESM.